doc/guides/writing-tests.md

```diff
@@ -18,6 +18,13 @@ Add tests when:
 - Fixing regressions and bugs.
 - Expanding test coverage.
 
+## Test directory structure
+
+See [directory structure overview][] for outline of existing test & locations,
+When deciding on whether to expand an existing test file or create a new one.
```

Comment:
Sorry for not being clear, I meant the punctuation at the end of this line should be `,`, so it should be

```
See [directory structure overview][] for outline of existing test & locations.
When deciding on whether to expand an existing test file or create a new one,
consider going through the files related to the subsystem.
```

Reply:
`When deciding on whether to expand an existing test file or create a new one,
consider going through the files related to the subsystem.`
@joyeecheung it will exceed 80 characters.

Reply:
I think @joyeecheung meant it to be a text block (without exceeding 80 characters by using new lines in the correct places), not as being in a single line.

Reply:
ok got it.